states/massachusetts/elections.toml

```diff
@@ -8,7 +8,87 @@ date = 2020-09-01
 name = "Congressional Primary"
 original_date = 2020-09-01
 
+[20200901.registration]
+in_person_by = 2020-08-22
+online_by = 2020-08-22T23:59:00
+postmarked_by = 2020-08-22
+sources = [
+  "http://www.sec.state.ma.us/ele/eleifv/howreg.htm",
+  "https://www.sec.state.ma.us/ele/elepdf/2020-Primary-and-Election-Schedule-Legal-Sized.pdf",
+]
+# Note the Day to Register was extended to 10 days from 20 days prior in 2020
+
+[20200901.absentee.application]
+email_by = 2020-10-31T12:00:00
+fax_by = 2020-10-31T12:00:00
+received_by = 2020-10-31T12:00:00
+# Because you can vote absentee in-person day before is OK
```

Comment:
I don't see where it says you can vote absentee in person. I only see in person early voting and this page (http://www.sec.state.ma.us/ele/eleev/early-voting-in-person.htm) says "You may vote at any early voting location in the city/town where you are registered to vote. " which implies there is no extra application.

Reply:
http://www.sec.state.ma.us/ele/eleabsentee/absidx.htm
Under the heading  "Voting"

> "If you would like to vote in person, you may do so in the office of your local election official. In-person absentee voting should be arranged ahead of time."

Reply:
padded dates